sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateClientTest.java

```diff
@@ -437,6 +429,28 @@ public void listCertificates(HttpClient httpClient, CertificateServiceVersion se
         });
     }
 
+
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("getTestParameters")
+    public void listPropertyOfCertificates(HttpClient httpClient, CertificateServiceVersion serviceVersion) {
```

Comment:
I would rename this to `listPropertiesOfCertificates` since that is the method name being called.